LOGGING/src/ILogger/ApplicationInsightsLogger.cs

```diff
@@ -168,23 +168,23 @@ private void PopulateTelemetry<TState>(ISupportProperties telemetryItem, TState
                 dict["EventName"] = eventId.Name;
             }
 
-            if (this.applicationInsightsLoggerOptions.IncludeScopes)
+            if (state is IReadOnlyCollection<KeyValuePair<string, object>> stateDictionary)
```

Comment:
this logic is moved out of the If(scopeenabled)  - thats the only change in this PR :)